### PR TITLE
Copy updates for ceph/what-is-ceph

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Check /ceph/what-is-ceph against [copy docs](https://docs.google.com/document/d/1gRuOAg6ZFBp-ikMq-DQBgQ4-BrXGeWOOe2cBxD3ohjE/edit#)

You'd have to check all the content, because the comments have been removed

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/10969

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/ceph/what-is-ceph.html [hunk 1]
```diff
@@ -33,307 +33,341 @@ <h1>
...
-            </div>
-          </div>
-        </section>
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Ceph features
+      </h2>
+      <ul class="p-list__item is-split">
```
Comment (minkyngkm) [inline templates/ceph/what-is-ceph.html:129]:
```suggestion
  <div class="row">
    <div class="col-8">
      <h2>
        Ceph features
      </h2>
    </div>
  </div> 
  <div class="u-fixed-width">  
      <ul class="p-list__item is-split">
```

File: templates/ceph/what-is-ceph.html [hunk 2]
```diff
@@ -33,307 +33,341 @@ <h1>
...
+    <div class="row">
+      <div class="col-10 u-sv3">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/b05b5a3c-Ceph_diagrams-02.svg",
+          alt="",
+          height="318",
+          width="661",
+          hi_def=True,
+          loading="auto|lazy"
```
Comment (minkyngkm) [inline templates/ceph/what-is-ceph.html:85]:
```suggestion
          loading="lazy"
```

File: templates/ceph/what-is-ceph.html [hunk 3]
```diff
@@ -33,307 +33,341 @@ <h1>
...
-            <p>
-              Ceph stores data as objects within logical storage pools. In order to achieve scalability, rebalancing and recovery capabilities, Ceph shards the pools into placement groups. The CRUSH algorithm then defines the placement group for storing an object and thereafter calculates which Ceph OSD Daemon should store the placement group.
-            </p>
-          </div>
-        </div>
-      </section>
+<section class="p-strip">
+  <div class="row">
+    <div class="row u-equal-height">
+      <h2>What is a Ceph cluster?</h2>
```
Comment (minkyngkm) [inline templates/ceph/what-is-ceph.html:74]: Would it be better if there is some space between the title and the image?

File: templates/ceph/what-is-ceph.html [hunk 4]
```diff
@@ -33,307 +33,341 @@ <h1>
...
-        </div>
+      <div class="col-10 u-sv3">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/9866ad8e-Ceph_diagrams-01.svg",
+          alt="",
+          height="393",
+          width="661",
+          hi_def=True,
+          loading="auto|lazy"
```
Comment (minkyngkm) [inline templates/ceph/what-is-ceph.html:50]:
```suggestion
          loading="auto"
```

**Review by minkyngkm — COMMENTED** [review]
Looks good @carkod! I made some minor suggestions and It seems the text needs to be updated as well In the line 18.

File: templates/ceph/what-is-ceph.html (hunk 1 above)
Comment (carkod) [reply templates/ceph/what-is-ceph.html]: I don't see the difference but done!

File: templates/ceph/what-is-ceph.html (hunk 4 above)
Comment (carkod) [reply templates/ceph/what-is-ceph.html]: Done, thanks.

File: templates/ceph/what-is-ceph.html (hunk 3 above)
Comment (carkod) [reply templates/ceph/what-is-ceph.html]: Added

**Comment by carkod** [comment]
> Looks good @carkod! I made some minor suggestions and It seems the text needs to be updated as well In the line 18.

Updated line 18. Thanks!

File: templates/ceph/what-is-ceph.html [hunk 5]
```diff
@@ -33,307 +33,344 @@ <h1>
...
+          <li class="p-list__item is-ticked">
+            Object level key-value mappings
+          </li>
+          <li class="p-list__item is-ticked">
+            Swift and AWS S3 Object API Compatibility
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
```
Comment (minkyngkm) [inline templates/ceph/what-is-ceph.html:156]: The `div` with the class `row` needs to be closed before the `div` with the class `u-fixed-width` It makes a small difference. Please update it if you think it looks better! 
image
 
image

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>
        Ceph features
      </h2>
    </div>
  </div>
  <div class="u-fixed-width">
    <ul class="p-list__item is-split">
        <li class="p-list__item is-ticked">
        Thin provisioning of block storage for disk usage optimisation
        </li>
        <li class="p-list__item is-ticked">
        Partial or complete read and writes and atomic transactions
        </li>
        <li class="p-list__item is-ticked">
        Replication and erasure coding for data protection
        </li>
        <li class="p-list__item is-ticked">
        Snapshot history, cloning and layering support
        </li>
        <li class="p-list__item is-ticked">
        POSIX file system semantics support
        </li>
        <li class="p-list__item is-ticked">
        Object level key-value mappings
        </li>
        <li class="p-list__item is-ticked">
        Swift and AWS S3 Object API Compatibility
        </li>
    </ul>
  </div>
```

**Review by minkyngkm — APPROVED** [review]

File: templates/ceph/what-is-ceph.html (hunk 5 above)
Comment (carkod) [reply templates/ceph/what-is-ceph.html]: Done

**Pull request merged**